settings.gradle.kts

```diff
@@ -0,0 +1,33 @@
+pluginManagement {
+  plugins {
+    // Default plugin versions
+    val indraVersion = "2.0.5"
+    id("net.kyori.indra") version indraVersion
+    id("net.kyori.indra.checkstyle") version indraVersion
+    id("net.kyori.indra.license-header") version indraVersion
+    id("net.kyori.indra.publishing.sonatype") version indraVersion
+    id("com.adarshr.test-logger") version "3.0.0"
+  }
+  includeBuild("build-logic")
+}
+
+rootProject.name = "adventure-parent"
+
+// Make sure to update bom/build.gradle when making changes to modules.
+
+sequenceOf(
```

Comment:
There was a reason these (include + projectdir) were kept separate - dependabot had a hard time detecting the module names. Is this still the case?